CHANGELOG.md

```diff
@@ -31,6 +31,16 @@ Only use disabled form controls if research shows it makes the user interface ea
 
 This was added in [pull request #3187: Add disabled styles for form controls](https://github.com/alphagov/govuk-frontend/pull/3187).
 
+#### Stop the Accordion from remembering and restoring sessions
```

Comment:
A thought – we might want to reword this as we already have a few other entries in the changelog that start with stop where we _want_ people to stop doing something, but in this instance we're just _allowing_ people to disable it where it's not desired behaviour:

- [We want you to] stop using the deprecated IE8 mixins and settings
- [We want you to] stop using the `govuk-button--disabled` class on buttons
- [You now have the option to] stop the Accordion from remembering and restoring sessions

Maybe something like 'Configure whether the Accordion remembers and restores sessions'?

Reply:
Makes sense to me! 

I'm working under the assumption that the changelog is going to get tidied up a bit pre-release anyway, but no harm in making things clearer now.